renovate.json5

```diff
@@ -9,5 +9,208 @@
     ],
     "useBaseBranchConfig": "merge",
     "branchPrefix": "renovate/hivemq-edge/",
-    "prConcurrentLimit": 20
+    "prConcurrentLimit": 20,
+    "packageRules": [
+        {
+            "matchPackagePatterns": [
+                "^io\\.confluent[.:]",
+                "^org\\.apache\\.kafka[.:]"
+            ],
+            "registryUrls": [
+                "https://packages.confluent.io/maven"
+            ]
+        },
+        {
+            "matchPackagePrefixes": [
+                "com.hivemq.commons:"
+            ],
+            "registryUrls": [
+                "https://maven.pkg.github.com/hivemq/hivemq-commons"
+            ]
+        },
+        {
+            "matchPackagePrefixes": [
+                "com.byteowls:"
+            ],
+            "registryUrls": [
+                "https://maven.pkg.github.com/hivemq/vaadin-chartjs"
+            ]
+        },
+        {
+            "matchPackagePatterns": [
+                "^com\\.github\\.everit-org\\.json-schema[.:]"
+            ],
+            "registryUrls": [
+                "https://jitpack.io"
+            ]
+        },
+        {
+            "groupName": "all patch dependencies",
+            "groupSlug": "all-patch",
```

Comment:
D o we want to keep the all-patches ?

Reply:
i split them up, I think we still want the patch dependencies that can be applied automotically